File: components/ai_chat/resources/untrusted_conversation_frame/components/assistant_response/test_utils.ts
// Copyright (c) 2025 The Brave Authors. All rights reserved.
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this file,
// You can obtain one at https://mozilla.org/MPL/2.0/.

/**
 * Shared test utilities for assistant response components
 */

export const setupMemoryToolStringConstants = () => {
  global.S = {
    ...(global.S || {}),
    CHAT_UI_MEMORY_UPDATED_WITH_CONTENT_LABEL:
      'CHAT_UI_MEMORY_UPDATED_WITH_CONTENT_LABEL',
    CHAT_UI_MEMORY_UNDO_BUTTON_LABEL:
      'CHAT_UI_MEMORY_UNDO_BUTTON_LABEL',
    CHAT_UI_MEMORY_MANAGE_ALL_BUTTON_LABEL:
      'CHAT_UI_MEMORY_MANAGE_ALL_BUTTON_LABEL',
    CHAT_UI_MEMORY_ERROR_LABEL: 'CHAT_UI_MEMORY_ERROR_LABEL',
    CHAT_UI_MEMORY_UNDONE_LABEL: 'CHAT_UI_MEMORY_UNDONE_LABEL'
  }
}

File: components/email_aliases/test_utils.cc
// Copyright (c) 2025 The Brave Authors. All rights reserved.
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this file,
// You can obtain one at https://mozilla.org/MPL/2.0/.

#include "brave/components/email_aliases/test_utils.h"

#include "brave/components/email_aliases/email_aliases_endpoints.h"

namespace email_aliases::test {

GURL GetEmailAliasesServiceURL() {
  return endpoints::GenerateAlias::URL();
}

}  // namespace email_aliases::test
